app/services/submission/SubmissionService.scala

```diff
@@ -107,22 +107,40 @@ class SubmissionService @Inject() (
   implicit val unloadingRemarkReads: Reads[UnloadingRemarkType] = {
     import pages._
 
-    for {
-      unloadingCompletion <- UnloadingTypePage.path.read[UnloadingType].map(unloadingTypeToFlag)
-      unloadingDate       <- DateGoodsUnloadedPage.path.read[LocalDate].map(localDateToXMLGregorianCalendar)
-      unloadingRemark     <- UnloadingCommentsPage.path.readNullable[String]
-      stateOfSeals        <- __.read[StateOfSeals].map(_.value)
-      conform <- stateOfSeals match {
-        case Some(false) => false: Reads[Boolean]
-        case _           => AddTransitUnloadingPermissionDiscrepanciesYesNoPage.path.read[Boolean].map(!_)
+    def generateUnloadingRemarkForRevisedProcedureYes: Reads[UnloadingRemarkType] =
+      dateTimeService.currentDateTime.toLocalDate.map(localDateToXMLGregorianCalendar).map {
+        unloadingDate =>
+          UnloadingRemarkType(
+            conform = Number1,
+            unloadingCompletion = Number1,
+            unloadingDate = unloadingDate,
+            stateOfSeals = Some(Number1),
+            unloadingRemark = None
+          )
```

Comment:
Think this can just be?

```
def generateUnloadingRemarkForRevisedProcedureYes: Reads[UnloadingRemarkType] =
  UnloadingRemarkType(
    conform = Number1,
    unloadingCompletion = Number1,
    unloadingDate = dateTimeService.currentDateTime.toLocalDate,
    stateOfSeals = Some(Number1),
    unloadingRemark = None
)
```